### PR TITLE
Fix message size e2e tests

### PR DESCRIPTION
**Pull request by kian-thompson**
The flakiness of these tests was caused by the large messages being sent and processed. Service or the client wouldn't be able to keep up fast enough consistently to handle these large ops, thus causing these tests to timeout. By reducing the size of the ops being sent, and subsequently the compression/chunking limits, these tests should be much less flaky.

[AB#2964](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/2964)
[AB#8330](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/8330)
[AB#7883](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/7883)
[AB#7924](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/7924)
[AB#10974](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/10974)
[AB#10975](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/10975)

### CODE REVIEW

**Comment by rajatch-ff** [comment]
> 		describe(`Large payloads (exceeding the 1MB limit) - ${

this is no longer true in this test case, right?

---
Refers to: packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts:324 in c9e8c34. [](commit_id = c9e8c349ee1efe6efa6774d1a2c8526bf8509e24, deletion_comment = False)

File: packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts [hunk 1]
```diff
@@ -321,31 +321,34 @@ describeCompat("Message size", "NoCompat", (getTestObjectProvider, apis) => {
 			},
 		).timeout(chunkingBatchesTimeoutMs);
 
-		describe(`Large payloads (exceeding the 1MB limit) - ${
+		describe(`Large payloads - ${
```
Comment (rajatch-ff) [inline packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts:324]: So do we not need tests that test for >1MB payload?

File: packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts [hunk 2]
```diff
@@ -321,31 +321,34 @@ describeCompat("Message size", "NoCompat", (getTestObjectProvider, apis) => {
...
 			enableGroupedBatching ? "grouped" : "regular"
 		} batches`, () => {
 			describe("Chunking compressed batches", () =>
 				[
-					{ messagesInBatch: 1, messageSize: 2 * 1024 * 1024 }, // One large message
-					{ messagesInBatch: 3, messageSize: 2 * 1024 * 1024 }, // Three large messages
-					{ messagesInBatch: 1500, messageSize: 4 * 1024 }, // Many small messages
+					{ messagesInBatch: 1, messageSize: 51 * 1024 }, // One large message (51 KB each)
```
Comment (rajatch-ff) [inline packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts:329]: consider defining a const kb = 1024 at the top once and re-use across the test suite. Will make it more readable.

File: packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts [hunk 3]
```diff
@@ -253,6 +253,7 @@ describeCompat("Message size", "NoCompat", (getTestObjectProvider, apis) => {
 		},
 	);
 
+	// See ADO:8608
```
Comment (rajatch-ff) [inline packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts:256]: Can you elaborate in PR description, what flakiness is caused by existing message size tests, and how does bringing that down make the tests more reliable?

**Review by rajatch5 — APPROVED** [review]

File: packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts (hunk 1 above)
Comment (kian-thompson) [reply packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts:324]: There are still a couple tests that verify simple behavior of sending large payloads (rejected vs accepted)

**Review by jatgarg — APPROVED** [review]

File: packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts (hunk 2 above)
Comment (rajatch-ff) [reply packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts]: What was the resolution here?

Comment (kian-thompson) [reply packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts]: Added a const `bytesPerKB`

**Pull request merged**